### PR TITLE
Added security policy and code of conduct.

### LINKED ISSUE #9: Add CODE_OF_CONDUCT

### LINKED ISSUE #10: Add Security Policy
Add a Security Policy to this Repository.

### PR DESCRIPTION
**Pull request by Abhinav-ark**
Resolves #9 #10

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Reporting a Vulnerability
+
+The Amrita PYQ team and community take all security vulnerabilities
+seriously. Thank you for improving the security of our open source 
+software. We appreciate your efforts and responsible disclosure and will
+make every effort to acknowledge your contributions.
+
+Report security vulnerabilities by emailing the Amrita PYQ team at <a href="mailto:abhinavramki2@gmail.com">abhinavramki2@gmail.com</a> or
```
Comment (Ashrockzzz2003) [inline SECURITY.md:20]: Creating issues and tagging us here is enough?

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]

**Comment by Ashrockzzz2003** [comment]
Also @Abhinav-ark 
Can you add these files inside the `.github` directory? It will make it easy to isolate from project files.
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

**Comment by Abhinav-ark** [comment]
> Also @Abhinav-ark Can you add these files inside the `.github` directory? It will make it easy to isolate from project files. https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

This will be shown in repo home page aside readme only if it is in root folder. Contributing.md we can move to docs directory.
image

File: SECURITY.md (hunk 1 above)
Comment (Abhinav-ark) [reply SECURITY.md:20]: Isn't it preferrable to report security vulnerabilities privately? Publicly announcing them might expose security vulnerabilities to parties with malicious intent who had no idea about them in the first place.

**Review by Ashrockzzz2003 — APPROVED** [review]
LGTM

**Pull request merged**